indico/modules/rb/client/js/util.jsx

```diff
@@ -273,7 +276,24 @@ export function renderRecurrenceWeekdays(weekdays) {
     type: 'conjunction',
   }).format(sortedWeekdays.map(weekday => weekdaysMap[weekday]));
 
-  return formattedWeekdays;
+  // only show the repetition if it's greater than 1
+  if (repetition && repetition > 1) {
+    return PluralTranslate.string(
+      'Every {repetition} week on {weekdays}',
```

Comment:
I think the singular form does not need the placeholder:
```suggestion
      'Every week on {weekdays}',
```